drivers/note/note_driver.c

```diff
@@ -592,8 +569,7 @@ static void note_record_taskname(pid_t pid, FAR const char *name)
   ti = (FAR struct note_taskname_info_s *)
         &g_note_taskname.buffer[g_note_taskname.head];
   ti->size = tilen;
-  ti->pid[0] = pid & 0xff;
-  ti->pid[1] = (pid >> 8) & 0xff;
+  ti->pid = pid;
```

Comment:
Ditto

Reply:
Here is the normal assignment of value to pid